packages/medusa/src/loaders/express.ts

```diff
@@ -22,17 +22,17 @@ export default async ({ app, configModule }: Options): Promise<Express> => {
     sameSite = "none"
   }
 
-  const { cookie_secret } = configModule.projectConfig
+  const { cookie_secret, session_options } = configModule.projectConfig
   const sessionOpts = {
-    resave: true,
-    saveUninitialized: true,
+    resave: session_options?.resave ?? true,
```

Comment:
I believe `resave` can be false by default.

Reply:
Added this to not "break" any existing session management, even though it is highly unlikely that any businesses actually use `resave`